spdx-utils/src/main/resources/simple-license-mapping.yml

```diff
@@ -79,8 +79,8 @@ mit-licensed: MIT
 mpl-2: MPL_2_0
 mpl2.0: MPL_2_0
 mpl2: MPL_2_0
-psf: PYTHON_2_0
-psfl: PYTHON_2_0
+psf: PSF-2.0
+psfl: PSF-2.0
```

Comment:
These have to map to the `PSF_2_0` enum.

Reply:
Fixed